Websites/webkit.org/commit-review.md

```diff
@@ -44,10 +44,10 @@ Inactive Committers can regain Active Committer status by landing (via the Commi
 
 Inactive Reviewers need to show that they are making an effort to get familiar with the changes that have happened in the project since they were last active by landing at least 3 non-trivial patches. Once they have landed the patches, they need to send an email requesting reactivation to webkit-reviewers. This request needs the support of 2 Active Reviewers to be granted.
 
-Note that regardless of a Committer or Reviewer's activity status, any subversion account that has not been used in the past year will be deactivated for security purposes. For example, a Reviewer that has reviewed a patch in the past year but has not committed may have their subversion account deactivated. To reactivate a deactivated subversion account, an Active Committer or Active Reviewer can send an email to webkit-committers requesting it.
+Note that regardless of a Committer or Reviewer's activity status, any account that has not been used in the past year will be deactivated for security purposes. For example, a Reviewer that has reviewed a patch in the past year but has not committed may have their account deactivated. To reactivate a deactivated account, an Active Committer or Active Reviewer can send an email to webkit-committers requesting it and access will be granted in 5 business days unless someone objects.
```

Comment:
This is quite misleading. We cannot and don't deactivate GitHub accounts. I think (but not quite certain) that we only update `contributors.json`.

Also, "security purposes" arguably no longer applies, as Microsoft takes care of that.

Reply:
I think "security purposes" still applies for reasons I will not go into here.